onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorRecurrentNeuralNetwork.cpp

```diff
@@ -182,6 +182,23 @@ class DmlOperatorRecurrentBase: public DmlOperator, public RecurrentHelper
         }
     }
     
+    bool ActivationNameCompare(const std::string& activationName, const char* attrValue)
```

Comment:
```suggestion
    bool ActivationNameCompare(std::string_view activationName, std::string_view attrValue)
```
Prefer string_view where possible.